local_units/test_views.py

```diff
@@ -133,6 +134,16 @@ def test_detail(self):
         self.assertEqual(response.data["type_details"]["name"], "Code 0")
         self.assertEqual(response.data["type_details"]["code"], 0)
 
+    def test_updated_at_updated_by(self):
+        self.authenticate()
+        user = UserFactory.create()
+        local_units = LocalUnitFactory.create_batch(2, country=self.country, type=self.type, created_by=user, modified_by=user)
+        local_unit = LocalUnit.objects.filter(id__in=[unit.id for unit in local_units]).first()
+        response = self.client.get(f"/api/v2/local-units/{local_unit.id}/")
+        self.assertIsNotNone(response.data["modified_by_details"]["id"])
```

Comment:
This should do the work rather than checking None.

```suggestion
        self.assertEqual(response.data["modified_by_details"]["id"], user.id)
```